node/pkg/processor/message.go

```diff
@@ -80,8 +78,7 @@ func (p *Processor) handleMessage(ctx context.Context, k *common.MessagePublicat
 	if p.logger.Core().Enabled(zapcore.DebugLevel) {
 		p.logger.Debug("observed and signed confirmed message publication",
 			zap.String("message_id", k.MessageIDString()),
-			zap.Stringer("txhash", k.TxHash),
-			zap.String("txhash_b58", base58.Encode(k.TxHash.Bytes())),
```

Comment:
might be worth keeping this. IIRC it was useful for debugging Solana (since their account keys and signatures are b58 encoded).